<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseDifficulty</class>
 <widget class="QWidget" name="chooseDifficulty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="easyButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hardButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mediumButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
